### PR TITLE
Edit dotnet article

### PR DESCRIPTION
**Pull request by Mxchaeltrxn**

### CODE REVIEW

File: _posts/dotnet/2020-02-08-setting-up-docfx-on-github-pages.md [hunk 1]
```diff
@@ -5,23 +5,24 @@ tags: [github-actions, c#, dotnet]
...
+Having clear documentation is important to the success of any kind of software project. For open source projects libraries, it reduces the amount of questions the maintainers need to respond to, increases adoption and makes it easier for others to contribute to the project.
 
-The guide details how you can create a free documentation website hosted on github pages using [Microsoft's open source DocFX](https://dotnet.github.io/docfx/index.html) library for a dotnet project (although it could easily be modified to support typescript/javascript libraries too. This tool will generate a static searchable website based on your markdown files and public exposed classes and members.
+This guide details how you can create a free documentation website hosted on github pages using [Microsoft's open source DocFX](https://dotnet.github.io/docfx/index.html) library for a dotnet project although it could easily be modified to support typescript/javascript libraries too. This tool generates a static searchable website based on your markdown files and public exposed classes and members.
+<!-- 'Public exposed classes and members sounds weird and I'm unsure what you're trying to express. -->
```
Comment (chinwobble) [inline _posts/dotnet/2020-02-08-setting-up-docfx-on-github-pages.md]: When you add comments to your classes and methods, automatic tools can read this metadata and generate html out of them like this 
https://www.npgsql.org/doc/api/Npgsql.html

This is called the API Reference or API Documentation.

Checking this is how you can easily learn about all the functionality of a library.

Comment (Mxchaeltrxn) [reply _posts/dotnet/2020-02-08-setting-up-docfx-on-github-pages.md]: I don't know how to make sure that, grammatically, public applies to both 'class' and method' in this kind of sentence. 

Public exposed sounds weird, public makes more sense to me and is also more concise. I also didn't know a member was a method (I think that's c# jargon)

Comment (chinwobble) [reply _posts/dotnet/2020-02-08-setting-up-docfx-on-github-pages.md]: How about `This tool generates API reference documentation based on your publicly exposed members such as classes and methods and their comments.`??

Comment (Mxchaeltrxn) [reply _posts/dotnet/2020-02-08-setting-up-docfx-on-github-pages.md]: Sounds fine to me

**Pull request merged**